test/evergreen.yml

```diff
@@ -2032,6 +2032,7 @@ tasks:
       - func: "fetch artifacts"
       - func: "cppsuite perf test"
         vars:
+          test_config: checkpoint_cleanup=(wait=60)
```

Comment:
```suggestion
          # Configure checkpoint cleanup to run more often as the test only runs for 90 seconds and checkpoint cleanup is triggered every 5 min. by default.
          test_config: checkpoint_cleanup=(wait=60)
```

Reply:
Fixed in 8ec30bb43357ceb882198e2c4051b595af1aafbb